Towny/src/main/java/com/palmergames/bukkit/towny/TownyEconomyHandler.java

```diff
@@ -250,22 +248,21 @@ public static boolean subtract(String accountName, double amount, World world) {
 	/**
 	 * Add funds to an account.
 	 * 
-	 * @param accountName account to add funds to
+	 * @param account the Account receiving money.
 	 * @param amount amount of currency to add
 	 * @param world name of world (for TNE Reserve)
 	 * @return true if successful
 	 */
-	public static boolean add(String accountName, double amount, World world) {
+	public static boolean add(Account account, double amount, World world) {
 
-		Player player = Bukkit.getServer().getPlayerExact(accountName);
-		Transaction transaction = new Transaction(TransactionType.ADD, player, amount);
+		Transaction transaction = new Transaction(TransactionType.ADD, null, account, amount);
 		TownyTransactionEvent event = new TownyTransactionEvent(transaction);
```

Comment:
I was thinking it might be better to move this logic to the Account class, that way we can have both accounts in the case of payTo (and we could maybe even add the reason to Transaction)

Reply:
For this I'm going to see if I can pull it back into the Account. It is probably more useful if the Transaction was more fleshed out.